commands/deployment_strategy.go

```diff
@@ -0,0 +1,75 @@
+package commands
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"github.com/cloudfoundry-incubator/multiapps-cli-plugin/util"
+	"strconv"
+)
+
+type DeploymentStrategy interface {
+	CreateProcessBuilder() *util.ProcessBuilder
+}
+
+type DeployCommandStrategy struct {
```

Comment:
Keep interface convention naming - DeployCommandDeploymentStrategy
Same with the below struct

Reply:
Done